command/helpers.go

```diff
@@ -713,3 +721,32 @@ func loadFromStdin(testStdin io.Reader) (string, error) {
 	}
 	return b.String(), nil
 }
+
+// ByteSize converts a number of bytes to a human readable string.
+func ByteSize(bytes uint64) string {
```

Comment:
I wonder if something like `ByteToHumanString` or similar would be better here? I feel the current name is a little ambiguous. It would also be nice to have some test cases to cover this new function.